lib/builder/webpack/utils/postcss.js

```diff
@@ -115,7 +115,7 @@ export default class PostcssConfig {
         this.preset = config.preset
         delete config.preset
       }
-      _.defaults(config, this.defaultConfig)
+      _.defaultsDeep(config, this.defaultConfig)
```

Comment:
This is not just for examples @clarkdo right?

Reply:
Yeah, this is only impacting change. How do you think ?

Reply:
If required we should make it as a Hotfix :)

Reply:
With this we can support : 

```js
    postcss: {
      plugins: [ ... ],
      preset: { ... }
    }
```

Reply:
Sure. let's consider it after 2